app/assets/stylesheets/plugins/wice_grid.scss

```diff
@@ -1,148 +1,81 @@
-.wg-detached-filter,
-.wice-grid {
+.wg-detached-filter, .wice-grid{
+
   .text-filter-container {
-    input {
-      display: inline;
+    input{
+      width:auto;
       margin-right: 10px;
-      width: auto;
+      display: inline;
     }
   }
 
-  a {
-    .date-label {
-      text-decoration: none;
-    }
-
-    .date-label:hover {
-      text-decoration: line-through;
-    }
+  thead th select {
+    display: inline-block;
   }
 
-  .clickable {
+  a.date-label {text-decoration: none;}
+  a.date-label:hover {text-decoration: line-through;}
+
+  .clickable{
     cursor: pointer;
     margin-bottom: 2px;
     margin-right: 2px;
   }
 
-  .ui-datepicker-trigger,
-  .wg-detached-filter .ui-datepicker-trigger {
+  .ui-datepicker-trigger, .wg-detached-filter .ui-datepicker-trigger{
```

Comment:
Each selector in a comma sequence should be on its own single line<br>Opening curly brace `{` should be preceded by one space